src/shared/test/listener-collection-test.js

```diff
@@ -19,7 +19,19 @@ describe('ListenerCollection', () => {
     assert.calledOnce(listener);
   });
 
-  it('unregisters event listeners', () => {
+  it('unregisters some event listeners', () => {
```

Comment:
Can you wrap the tests here in `describe(<method name>)` blocks to make it more obvious which functions are being tested, eg:

```js
describe('#remove', () => {
  it('removes an event listener', () => { ... });
});

describe('#removeAll', () => {
  it('unregisters all event listeners', () => { ... });
});
```